### PR TITLE
opensuse compile dependencies

### PR DESCRIPTION
**Pull request by rriemann**

### CODE REVIEW

**Comment by sr55** [comment]
Thanks for the contribution.

You'd need to add a menu entry as well in the table of contents file.

File: source/docs/en/latest/developer/install-dependencies-opensuse.markdown [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+- add Packman repositoryies:
+
+~~~
+sudo zypper ar -cfp 90 https://ftp.fau.de/packman/suse/openSUSE_Tumbleweed/Essentials packman-essentials
+sudo zypper ar -cfp 90 https://ftp.fau.de/packman/suse/openSUSE_Tumbleweed/Multimedia packman-multimedia
+~~~
+
+Dependencies:
+
+- autoconf
```
Comment (sr55) [inline source/docs/en/latest/developer/install-dependencies-opensuse.markdown:33]: you may want to provide full commands for these that people can copy and paste.

**Comment by rriemann** [comment]
I do not have the chance to test with a fresh opensuse system, but I think that the documentation is already as-is very useful. I added the commands for copy'n'paste.

**Comment by bradleysepos** [comment]
Superseded by #202.

**Pull request closed without merging**